scripts/run-ci-local.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I think both Tara and Tobias have been using the Windows CMD shell for development, in which case this script will not run.

We could document that bash is a prerequisite for UX Aspects contribution. But it might be a bit more friendly to reimplement this as a node script.